src/lib/Form/Type/ChoiceList/Loader/ContentTypeChoiceLoader.php

```diff
@@ -36,14 +37,19 @@ public function __construct(ContentTypeService $contentTypeService, UserLanguage
      */
     public function getChoiceList(): array
     {
-        $contentTypes = [];
+        $contentTypesList = [];
         $preferredLanguages = $this->userLanguagePreferenceProvider->getPreferredLanguages();
         $contentTypeGroups = $this->contentTypeService->loadContentTypeGroups($preferredLanguages);
         foreach ($contentTypeGroups as $contentTypeGroup) {
-            $contentTypes[$contentTypeGroup->identifier] = $this->contentTypeService->loadContentTypes($contentTypeGroup, $preferredLanguages);
+            $contentTypes = $this->contentTypeService->loadContentTypes($contentTypeGroup, $preferredLanguages);
+            usort($contentTypes, function (ContentType $contentType1, ContentType $contentType2) {
+                return strcasecmp($contentType1->getName(), $contentType2->getName());
```

Comment:
Maybe we might use https://www.php.net/manual/en/function.strnatcasecmp.php if there is equivalent on JavaScript side?